elementary/monitor/cli.py

```diff
@@ -352,6 +359,7 @@ def report(
     """
     Generate a local observability report of your warehouse.
     """
+    vars = yaml.loads(dbt_vars) if dbt_vars else None
```

Comment:
Can you rename it to `dbt_vars` or directly assign it as the argument?
This overrides the `vars` function.

Reply:
Yes, sounds great.